res/bnd/annotations-api.jar.tmp.bnd

```diff
@@ -15,6 +15,7 @@
 
 -include: build-defaults.bnd, spec-defaults.bnd
 
+Bundle-License: https://www.apache.org/licenses/LICENSE-2.0.txt
```

Comment:
Could all of these Bundle-License clauses simply be put into the build-detaults.bnd file or similar?